bootstrap/pesticide_seed.py

```
@@ -0,0 +1,1179 @@
+    return target
+
+
+def iter_pairs(raw_row: dict[str, Any]) -> list[tuple[str, str, str]]:
+    crop_names = split_top_level(raw_row.get("CROPS_NM")) or ["작물정보없음"]
+    target_names = split_top_level(raw_row.get("SICKNS_HLSCT_NM_WEEDS_NM")) or [
+        "대상정보없음"
+    ]
+    pairs: list[tuple[str, str, str]] = []
+    for crop_name in crop_names:
+        for target_name in target_names:
+            pairs.append((crop_name, target_name, infer_target_kind(target_name)))
```

Ruff (0.15.12)

[warning] Use `list.extend` to create a transformed list

(PERF401)